example/satosa/pyeudiw_backend.yaml

```diff
@@ -265,8 +266,7 @@ config:
       module: pyeudiw.trust.handler.x509
       class: X509Handler
       config:
-        # client_id: *client_id
-        client_id_scheme: x509_san_dns # this will be prepended in the client id scheme used in the request. 
+        client_id: 'x509_san_dns:example.com'
```

Comment:
is the x509 chain about this RP configured to example.com?

if not, the init of the backed should check this and stop the execution